pkg/schemes/metallb/frrtypes/frr_node_state_types.go

```diff
@@ -0,0 +1,44 @@
+package frrtypes
```

Comment:
Please use sync script (refer to README) and open separate PR for this change.

Reply:
Ran the sync script for frr.